compose-builder/mqtt-bus.env

```diff
@@ -1,4 +1,3 @@
 MESSAGEQUEUE_TYPE=mqtt
 MESSAGEQUEUE_PROTOCOL=tcp
-MESSAGEQUEUE_HOST=edgex-mqtt-broker
```

Comment:
This change is not needed. Only services using this file should be the ones that need this also.